creepy/memory.py

```diff
@@ -3,16 +3,17 @@
 
 MCL_CURRENT = 1
 MCL_FUTURE  = 2
+
 _libc = ctypes.CDLL('libc.so.6', use_errno=True)
 
 
-def mlockall(flags=MCL_CURRENT|MCL_FUTURE):
```

Comment:
Maybe IntEnum would be better?

Reply:
Here we replicate part of libc interface and it's more convenient to keep all names the same as in libc. If we use different names then it would be harder to google and match libc functions/constants with ours.